plugins/backstage-plugin-env0/src/components/env0-deployment-table/env0-deployment-table.tsx

```diff
@@ -22,6 +24,14 @@ const getFormattedDeploymentDuration = (deployment: Deployment) => {
   return parseTimerElapsedTime(durationInSeconds);
 };
 
+const EllipsisTypography = styled(Typography)(() => ({
+  display: 'block',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  width: '20em',
```

Comment:
let me know if you have better idea other than this const width,
without it the column width is unlimited

Reply:
You can't set the width from the parent container? Since the column sizes should be set there 🤔